package.json

```
@@ -8,20 +8,33 @@
     "./test/utils/create-libp2p-node": false,
     "./test/utils/create-temp-repo-nodejs.js": "./test/utils/create-temp-repo-browser.js"
   },
+  "types": "dist/src/index.d.ts",
+  "typesVersions": {
+    "*": {
+      "src/*": [
+        "dist/src/*",
+        "dist/src/*/index"
+      ]
+    }
+  },
+  "eslintConfig": {
+    "extends": "ipfs"
+  },
   "files": [
     "dist",
     "src"
   ],
   "scripts": {
+    "prepare": "aegir build",
```

We don't need full build here

```suggestion
    "prepare": "aegir ts -p check",
```